SwiftUIDZ/SwiftUIDZ/CartView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  CartView.swift
+//  SwiftUIDZ
+//
+//  Created by Анна Сычева on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Экран корзины
+struct CartView: View {
+    
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let choosedText = "You choosed"
+        static let daysText = "days"
+        static let rubText = "RUB"
+        static let payButtonTitle = "Pay"
+        static let spacing: CGFloat = 10
+        static let buttonWidth: CGFloat = 100
+        static let buttonHeight: CGFloat = 50
+        static let cornerRadius: CGFloat = 8
+        static let alertTitle = "Congratulations"
+    }
+    
+    // MARK: - Public Properties
+    
+    @EnvironmentObject var subscriptionOptions: SubscriptionOptions
```

Comment:
```suggestion
    @EnvironmentObject private var subscriptionOptions: SubscriptionOptions
```
Вроде бы это тоже можно сделать приватным